### ISSUE TITLE
won't run - "undefined method"?

### ISSUE THREAD

**Issue by ghost** [bachya/Sifttter-Redux#1]

**Comment by ghost** [comment]
I get this error in Terminal when running Sifttter-redux for the first time on a Mac Mini (Mavericks):

```
$ srd help
/Library/Ruby/Gems/2.0.0/gems/sifttter-redux-0.3.6/lib/sifttter_redux/sifttter.rb:8:in `<module:Sifttter>': undefined method `using' for SifttterRedux::Sifttter:Module (NoMethodError)
from /Library/Ruby/Gems/2.0.0/gems/sifttter-redux-0.3.6/lib/sifttter_redux/sifttter.rb:7:in `<module:SifttterRedux>'
from /Library/Ruby/Gems/2.0.0/gems/sifttter-redux-0.3.6/lib/sifttter_redux/sifttter.rb:1:in `<top (required)>'
from /System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/lib/ruby/2.0.0/rubygems/core_ext/kernel_require.rb:45:in `require'
from /System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/lib/ruby/2.0.0/rubygems/core_ext/kernel_require.rb:45:in `require'
from /Library/Ruby/Gems/2.0.0/gems/sifttter-redux-0.3.6/lib/sifttter_redux.rb:6:in `<top (required)>'
from /System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/lib/ruby/2.0.0/rubygems/core_ext/kernel_require.rb:45:in `require'
from /System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/lib/ruby/2.0.0/rubygems/core_ext/kernel_require.rb:45:in `require'
from /Library/Ruby/Gems/2.0.0/gems/sifttter-redux-0.3.6/bin/srd:39:in `<top (required)>'
from /usr/bin/srd:23:in `load'
from /usr/bin/srd:23:in `<main>'
```

**Comment by bachya** [comment]
Fixed and pushed to version 0.3.7.
